### ISSUE TITLE
Disable Game Config

### ISSUE THREAD

**Issue by SyndiShanX** [jaredlll08/WaitingTime#9]
It would be nice to add a Config Option to disable loading the game, but keeping the stats

**Comment by ghost** [comment]
I agree, this would be useful for those who still want the load time stats but don't care for pong as much.

I would also like to suggest an option to allow the stats to bypass the disabling of the forge splash so that the stats still show.

**Comment by Abcity2** [comment]
Is there a way to reset the stats?
